lifecycle_msgs/msg/Transition.msg

```diff
@@ -44,6 +44,14 @@ uint8 TRANSITION_ON_ERROR_SUCCESS = 60
 uint8 TRANSITION_ON_ERROR_FAILURE = 61
 uint8 TRANSITION_ON_ERROR_ERROR = 62
 
+# These transitions are internally
+# available to the lifecycle node to
+# be called when an error arises during
+# normal operation that causes the node
+# to need reconfiguration.
+uint8 TRANSITION_ACTIVE_ERROR = 63
```

Comment:
These should probably not be sequential but pick a new decade starting with 70

Reply:
https://github.com/ros2/rcl_interfaces/blob/018fe20d92423828e0dd629eb96759ec033c2fef/lifecycle_msgs/msg/Transition.msg#L43
> Reserved [10-69], private transitions

it needs to expand reserved range.